README.md

```diff
@@ -35,9 +35,9 @@ These are renderers that you can toggle on anytime. Each of them comes with thei
 - ### GL4ES
 GL4ES (**[OpenGL ES](https://www.khronos.org/opengles/#)**) has moderate speed, with moderate mod compatibility. Complex mods like Distant Horizons and Immersive Portals may not work.
 - ### Zink
-**[Zink](https://docs.mesa3d.org/drivers/zink.html)**, while being primarily slow, has great mod compatibility. Complex mods like Distant Horizons and Immersive Portals will work.
+**[Zink](https://docs.mesa3d.org/drivers/zink.html)**, while being primarily slow, has great mod compatibility. Complex mods like Distant Horizons and Immersive Portals shoiuld work.
```

Comment:
You made a spelling mistake here, BK lover.

Reply:
Might need to check your spelling again bud

Reply:
Who me?

Reply:
![image](https://user-images.githubusercontent.com/87071118/229374032-4cbf7689-2c36-42ad-9749-4863fe5e80c8.jpeg)

Reply:
I'm confused. I didn't put that mistake there, you did.